### PR TITLE
Example app latency re-run, Minor doc updates, Add grunt to devDependencies

### PR DESCRIPTION
**Pull request by kmix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,51 +14,51 @@ For deploying to a test server, see the README under **ansible** folder.
 
 ##Client Server Setup
 The application consists of a node/express software running the server and plain javascript/html on the client. The server
-powers reset endpoints, websocket communication, database interactions and the client website. 
+powers rest endpoints, websocket communication, database interactions and the client website.
```
Comment (kemarc) [inline README.md]: R.E.S.T.

**Comment by kemarc** [comment]
+1

File: README.md (hunk 1 above)
Comment (kmix) [inline README.md]: I've never seen it with periods, as each letter isn't an abbreviation of an individual word: 

https://en.wikipedia.org/wiki/Representational_state_transfer

Would all-caps work (REST)?

**Comment by amenjava** [comment]
Per conversation with kemarc:
2018-10-31 - kemarc [12:56 PM]
That's just for the example pages.
I'd drop it. It's been over a year.

**Pull request closed without merging**